chaco/api.py

```diff
@@ -161,6 +154,95 @@
 
 - :class:`~.AbstractController`
 
+Colormaps and Color Palettes
+----------------------------
+
```

Comment:
I would prefer adding a subsection here for color maps and another for color palettes

Reply:
I also added a "Utilities / Convenience Objects" subsection here as well